source/_integrations/miele.markdown

```diff
@@ -0,0 +1,155 @@
+---
+title: Miele
+description: Instructions on how to set up the Miele integration within Home Assistant.
+ha_category:
+  - Hub
+  - Sensor
+ha_iot_class: Cloud Push
+ha_release: '2025.5.0'
+ha_domain: miele
+ha_codeowners:
+  - '@astrandb'
+ha_config_flow: true
+ha_platforms:
+  - sensor
+ha_integration_type: integration
+---
+
+The Miele integration allows users to integrate their home appliances using the [official 3rd party API](https://www.miele.com/developer). The integration will make the best effort to map the data-points in the API to sensors, switches, etc. in Home Assistant.
```

Comment:
I think a bit more overview about what Miele is is nice

Reply:
Well, I copied this line from their corporate web and downgraded it somewhat.